### ISSUE TITLE
[FE] Severity 8: Web #7 - When the user activates "Talk to the Veterans Crisis Line now" control (either at the top or the bottom of the Screen) and the modal dialog appears,  visual focus is set to the "Call 988 and select 1" link,  while the screen reader reads out the entire contents of the dialog at once.  The consequence is that all text runs together,  and the SR does not announce the interactive elements,  such as which pieces of content are links.   

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#59317]
## Notes
LP: I'm guessing that we don't have any control over this control?

## Issue
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/86678742/8e167ecc-f09f-4fc6-a34f-6153275aab87)

Original Spreadsheet Source for Issue: [Mobile Audit 05102023 Patient Check-in (eCheck-in  Pre-Check-in) Web.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/11614848/Mobile.Audit.05102023.Patient.Check-in.eCheck-in.Pre-Check-in.Web.xlsx)

## Tasks
- [ ] [FE] Resolve issue
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations (readme and use cases)

## Acceptance Criteria
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Tested on staging
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature
- [ ] Prod validation:
  - [ ] not necessary
  - [ ] scheduled

## Code Review Steps
- [ ] Read ticket details and verify that ACs are met
- [ ] Review code in PR
- [ ] Run branch locally
- [ ] Check that new and updated translations use new keys
- [ ] Check for hardcoded strings

**Comment by lee-delarm6** [comment]
I don't believe we have control over this, design team will likely take this

**Comment by loripusey** [comment]
Submitted Platform Support ticket for the Design System team to pick up this issue:
https://dsva.slack.com/archives/CBU0KDSB1/p1687354617920519
